app/models/user.rb

```diff
@@ -25,6 +25,8 @@ def current_user? user
 
 
   def activate
+    update_attribute :activated, true
```

Comment:
không đúng rồi anh.
lúc trước mình phân tích cái update_attributes vs update_attribute thì có nói rằng: thằng update_attributes sẽ đảm bảo việc bảo toàn dữ liệu. vì vậy trong bất kì trường hợp update nào thì mình cũng sử dụng update_attributes mà